lib/rake/application.rb

```diff
@@ -604,5 +604,9 @@ def rakefile_location backtrace = caller
 
       backtrace.find { |str| str =~ re } || ''
     end
+
+  private
+    FIXNUM_MAX = (2**(0.size * 8 - 2) - 1) # :nodoc:
+    
```

Comment:
Or go straight to ruby/rake#82 for a more future-proof implementation of the constant!